src/daemon/SocketServer.cpp

```diff
@@ -181,6 +181,15 @@ namespace SDDM {
                 daemonApp->powerManager()->hybridSleep();
             }
             break;
+            case GreeterMessages::KeyboardLayout: {
+                qDebug() << "Requested Keyboard Layout switch";
```

Comment:
excessive debug

Reply:
Literally every switch case has a debug statement, what makes this one excessive?

Reply:
It was the only one visible in the diff :)

If the others have similar debugs, ignore this comment.